files/es/glossary/sql/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SQL
+slug: Glossary/SQL
+tags:
+  - BBDD
+  - Glosario
+  - Sql
```

Comment:
we are not adding tags anymore. 

```suggestion
```

Reply:
Ah, this was automatically added by the markdown conversion tool. I reckon we need to raise an issue about this or we will have to manually delete them. Thank you for highlighting this.

Reply:
thanks for testing with es files :+1: